src/Plugin/DocReader/PDFParserReader.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Drupal\dropai\Plugin\DocReader;
+
+use Smalot\PdfParser\Parser;
+
+class PDFParserReader implements DocReaderInterface {
+
+  public function getText($filePath) {
+    $content = '';
+    $parser = new Parser();
+
+    try {
+      $pdfDoc = $parser->parseFile($filePath);
+      $content = $pdfDoc->getText();
+    } catch (Exception $e) { }
```

Comment:
It's generally not a good practice to catch an exception and not do anything with it. I would at least write an error to the logs OR let this error bubble up to. whatever called it and catch the error there.